.github/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,30 @@
+To make sure that everyone has a common understanding of “show respect and courtesy to each other,” we have adopted the following code of conduct. The code of conduct is enforced by the Kindly core team.
+If someone has violated this code of conduct please report this to one of the lead developers or the product manager
+Unacceptable behaviour 
+
+The following types of behaviour are unacceptable for Kindly developers, and constitute code of conduct violations.
+Abusive behaviour 
+
+Harassment—including offensive verbal comments related to gender, sexual orientation, disability, physical appearance, body size, race, or religion, as well as sexual images in public spaces, deliberate intimidation, stalking, following, harassing photography or recording, inappropriate physical contact, and unwelcome sexual or romantic attention.
+Threats—threatening someone physically or verbally. For example, threatening to publicize sensitive information about someone’s personal life.
+Unwelcoming behaviour 
+
+Blatant discrimination—saying things that are explicitly racist, sexist, homophobic, transphobic etc. For example, arguing that some people are less intelligent because of their gender, race or religion. Subtle mistakes made in conversation are not code of conduct violations. However, repeating something after it has been pointed out to you that you broke a social rule, or antagonizing or arguing with someone who has pointed out your mistake is considered unwelcoming behaviour, and is not allowed.
+Maliciousness towards others—deliberately attempting to make others feel bad, name-calling, singling out others for derision or exclusion. For example, telling someone they’re not a real programmer or that they don’t belong on our project.
+Being extremely unpleasant—for example, if we’ve received reports from multiple people of annoying, rude, or especially distracting behaviour that is making it harder for others to work.
+
+##Scope 
+
+Members are held to the standards outlined in this code of conduct when interacting in our physical space, at our events, and in our Discord community, GitHub repositories, Zoom meetings or other online spaces.
+Abusive or unwelcoming behaviour between members still has a profound impact on individuals and on the community when it happens beyond our walls. The Kindly core team will use our discretion when deciding whether to enforce this code of conduct and potentially remove someone from the community after reports of such behaviour happening outside of the community, taking into account the impact on the individuals involved as well as the impact on the community and the public perception of our organisation.
+When in doubt, please report unacceptable behaviour to us. 
+Enforcement 
+
+We’ve categorized unacceptable behaviour into “abuse” and “unwelcoming behaviour” in the section above.
```

Comment:
No we haven't.

There's no categorisation of 'abuse' or 'unwelcoming behaviour' but there is a mention to 'Unwelcoming behaviour' (I'm glad you stuck to the European spelling 😄 )